frontend/src/features/workspace/components/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -52,11 +46,7 @@ export const WorkspaceHeader = ({
         direction={{ base: 'column', md: 'row' }}
         h="fit-content"
       >
-        <WorkspaceSortDropdown
-          value={sortOption}
-          onChange={setSortOption}
-          isDisabled={isLoading}
-        />
+        {/* Future: add form sorting dropdown here, can use WorkspaceSortDropdown component */}
```

Comment:
can probably delete the entirety of WorkspaceSortDropdown for now, no point keeping dangling code.

Reply:
okiee